doc/api/globals.md

```diff
@@ -592,8 +597,11 @@ added: v22.4.0
 
 A browser-compatible implementation of [`localStorage`][]. Data is stored
 unencrypted in the file specified by the [`--localstorage-file`][] CLI flag.
+The maximum amount of data that can be stored is 10 MB.
 Any modification of this data outside of the Web Storage API is not supported.
 Enable this API with the [`--experimental-webstorage`][] CLI flag.
+`localStorage` data is not stored per user or per request when used in the context
+of a server, it is shared across all users and requests.
```

Comment:
WDYT about this extra bit of information, is this implied?

Reply:
@cjihrig asked for [this.](https://github.com/nodejs/node/pull/53881#discussion_r1683331170)